### ISSUE TITLE
latest moar not honor filename tab completion 

### ISSUE THREAD

**Issue by ciscohack** [walles/moar#147]
I have noticed in recent moar update not able to do moar filename table completion. when I type moar and press tab it not list available test file to parse through moar.

**Comment by ciscohack** [comment]
I just reinstalled the moar and added export pager variable in zshrc file and it's started working but I didn't understand why it failed first placed

**Comment by walles** [comment]
TAB completion is 100% by the shell and 0% by `moar`.

Some zsh forum would be a better place to ask this.

**Comment by ciscohack** [comment]
@walles thanks for the comment. I don't know whether it is zsh issue or moar becoz if it is zsh issue I have bat also install and it is working fine. so the point is if this is zsh issue it will fail for all.

One more issue I have noticed with warp terminal where moar selection is not working .........I tried pressing shift/alt/option key none of the keyboard key help me to select the text ...moar need better keyboard and selection handling out of box.

Here I am using oh-my-fish shell
